groups/sig-release/groups.yaml

```diff
@@ -323,38 +323,40 @@ groups:
       - ramses.green.2@gmail.com # 1.29 Lead Shadow
       - xandergrzyw@gmail.com # 1.29 Emeritus Advisor
     members:
+      - abigailkmccarthy@gmail.com # 1.29 Communications Shadow
+      - AnaMMedina21@gmail.com # 1.29 Enhancements Shadow
+      - bradmccoydev@gmail.com # 1.29 Bug Triage Shadow
+      - drewhagendev@gmail.com # 1.29 Docs Shadow
+      - egbunaoluebube@gmail.com # 1.29 Docs Shadow
+      - faeka6@gmail.com # 1.29 Release Notes Shadow
       - fsmunoz@gmail.com # 1.29 Release Notes Lead
-      #- X # 1.29 Release Notes Shadow
-      #- X # 1.29 Release Notes Shadow
-      #- X # 1.29 Release Notes Shadow
-      #- X # 1.29 Release Notes Shadow
+      - harshitasao@gmail.com # 1.29 Docs Shadow
+      - hkamel.msft@gmail.com # 1.29 Communications Shadow
       - jackhammervyom@gmail.com # 1.29 CI Signal Lead
-      #- X # 1.29 CI Signal Lead Shadow
-      #- X # 1.29 CI Signal Lead Shadow
-      #- X # 1.29 CI Signal Lead Shadow
-      #- X # 1.29 CI Signal Lead Shadow
+      - james@quigley.us # 1.29 Communications Shadow
       - kat.cosgrove@gmail.com # 1.29 Docs Lead
-      #- X # 1.29 Docs Shadow
-      #- X # 1.29 Docs Shadow
-      #- X # 1.29 Docs Shadow
-      #- X # 1.29 Docs Shadow
+      - kcmartin2@mac.com # 1.29 Communications Shadow
+      - mankulka@redhat.com # 1.29 CI Signal Shadow
       - markrossetti@gmail.com # 1.29 Release Manager associate
       - marosset@microsoft.com # 1.29 Release Manager associate
+      - maryam_tavakkoli@hotmail.com # 1.29 Bug Triage Shadow
+      - mengjiao.liu@daoloud.io # 1.29 Release Notes Shadow
```

Comment:
```suggestion
      - mengjiao.liu@daocloud.io # 1.29 Release Notes Shadow
```

Reply:
@Priyankasaggu11929 could you push an update for these?

Reply:
Hey @salaxander, pushed the changes in the recent commit refresh. Will need another `lgtm` and `approve`. Thanks!